modules/mod_ginger_collection/templates/beeldenzoeker/record-meta.tpl

```diff
@@ -144,7 +152,7 @@
             <dl class="adlib-object__meta__data">
             {% if record['acquisition.date'] %}
                 <dt>{_ Acquired _}</dt>
-                <dd>{{ record['acquisition.date']|isodate:"j F Y" }}{% if record['acquisition.method'] %}, {{ record['acquisition.method'] }}{% endif %}</dd>
+                <dd>{% if record['acquisition.method'] %}, {{ record['acquisition.method'] }}{{ record['acquisition.date']|isodate:"j F Y" }}{% endif %}</dd>
```

Comment:
Shouldn’t this be the other way around: `method, date` instead of `, methoddate`?